### DISCUSSION TITLE
Looks like Insteon is dead

### DISCUSSION THREAD

**Discussion by ellucianBret** [kuestess/homebridge-platform-insteonlocal#253] [General]
Attempts to log in to their mobile app fail. Every page at insteon.com shows a vague message that they are gone.

Bret

**Comment by kuestess** [comment]
@ellucianBret Yep, they died about a month ago.  Plugin should still work fine.

**Reply by ellucianBret** [reply]
My hub is not working though

**Comment by 64Spaces** [comment]
@ellucianBret, there is a lot of information on this Reddit page: (https://reddit.com/r/Insteon). It blew up a month ago. So if you look at the history, you can see where others were affected. There are several paths that Insteon users are pursuing by accessing the Hub and gaining control again. InsteonLocal is one of the many paths. I have used InsteonLocal with my Hub for a couple of years. I ended up unplugging my Hub and redoing everything on a PLM. InsteonLocal is now running my HomeKit setup again. It takes a little research and then decisions about which way to go.
